packages/strapi-plugin-content-manager/config/functions/bootstrap.js

```diff
@@ -331,9 +331,10 @@ module.exports = async cb => {
 
     // Update other keys
     sameApis.map(apiPath => {
+
       // This doesn't keep the prevSettings for the relations,  the user will have to reset it.
       // We might have to improve this if we want the order of the relations to be kept
-      const keysToUpdate = ['relations', 'loadedModel', 'associations', 'attributes', ['editDisplay', 'relations']].map(key => apiPath.concat(key));
+      const keysToUpdate = ['relations', 'loadedModel', 'associations', 'attributes', ['editDisplay', 'relations'], ['editDisplay', 'fields']].map(key => apiPath.concat(key));
```

Comment:
Well by changing this it doesn't keep the previous layout of the `editPage` and also it keeps the previous relations of the content type.

I'm having a hard time reproducing the bug everything works as expected when using content types in lowercase.
I think it has something to do with `camelCase` ones or the one containing white spaces.

Reply:
Well with the modifications from above you don't need to modify this line.

Reply:
Removed line, rebased.